modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -1173,7 +1173,7 @@ protected function add_attributes_to_container( $container, $item_settings ) {
 
 	protected function render() {
 		$settings = $this->get_settings_for_display();
-		$widget_number = substr( $this->get_id_int(), 0, 3 );
+		$widget_number = $this->get_id_int();
```

Comment:
I have updated this use the full, unique identifier, instead of the abbreviated version with 3 integers.